RefactoringPracticeI/GameInventory/Inventory/HandlerAgedBrie.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace RefactoringExerciseI.Inventory
+{
+    public class HandlerAgedBrie : IHandlerItem 
+    {
+        private IHandlerItem? _nextHandler;
+
+        public IHandlerItem? NextHandler { get; }
+
+        public void SetNext(IHandlerItem NextHandler)
+        {
+            _nextHandler = NextHandler;
+        }
+
+        public void UpdateQuality (Item item)
```

Comment:
This method name could be misleading at it seems it not only updates quality but sellIn too